doc/tutorials/board_inspector_tool.rst

```diff
@@ -28,11 +28,11 @@ a board configuration file.
 #. Make sure the target system is set up and ready to run the Board Inspector,
    according to :ref:`gsg-board-setup` in the Getting Started Guide.
 
-#. Load the ``msr`` driver, used by the Board Inspector:
+#. Before using Board Inspector, you need to update the pciid information:
```

Comment:
Keep this one in sync with the other sentence if you change it based on my recommendation

Reply:
Same here:
```
#. On the target, you can optionally update the ``pci.ids`` file
   that contains the list of known identifiers for PCI devices:
```